wp-modules/api-data/api-data.php

```diff
@@ -108,6 +110,7 @@ function get_pattern_names() {
  */
 function delete_pattern( $request ) {
 	$is_success = \PatternManager\PatternDataHandlers\delete_pattern( $request->get_params()['patternName'] );
+	tree_shake_theme_images( get_wp_filesystem_api(), 'copy_dir' );
```

Comment:
Runs on deleting a pattern in the Patterns UI:

<img width="1183" alt="Screenshot 2023-06-13 at 1 19 55 AM" src="https://github.com/studiopress/pattern-manager/assets/4063887/41288af6-cf57-463e-928a-6baaaf867dec">

This has a lower risk of being a whack-a-mole bug.